pydisdrometer/ParsivelNasaGVReader.py

```diff
@@ -31,7 +45,7 @@ def read_parsivel_nasa_gv(filename, campaign='ifloods'):
     recalculate it based upon a fall speed relationship.
     '''
 
-    reader = NASA_APU_reader(filename, campaign)
+    reader = NASA_APU_reader(filename, campaign, skip_header)
 
```

Comment:
What do you think about leaving the skip_header for potential future changes of files?